CMakeLists.txt

```
@@ -61,6 +61,11 @@ if (NOT IGNORE_VENDORED_DEPENDENCIES)
   include(fmtlib)
   include(nlohmann-json)
   include(range-v3)
+else ()
+  message(WARNING "-- Ignoring vendored dependencies, found versions may differ. Make sure to thoroughly test resulting binaries before using them in production.")
```

I think it's important to mention running the test suite explicitly. I doubt anyone will really bother to test it manually but they might still be willing to at least run our tests :)

How about this?
```suggestion
  message(WARNING "-- Ignoring vendored dependencies. Will use installed versions if found. Versions may differ from the ones the compiler was tested with. Make sure to run the test suite and thoroughly test the resulting binaries before using them in production.")
```

---

Yeah that's good! Fixed :)